app/models/metric/ci_mixin.rb

```diff
@@ -21,12 +21,9 @@ module Metric::CiMixin
     end
   end
 
-  def has_perf_data?(interval_name = "hourly")
-    @has_perf_data ||= {}
-    unless @has_perf_data.key?(interval_name) # memoize boolean
-      @has_perf_data[interval_name] = associated_metrics(interval_name).exists?
-    end
-    @has_perf_data[interval_name]
+  def has_perf_data?
+    return @has_perf_data unless @has_perf_data.nil?
```

Comment:
Unfortunately, it can't.
```
irb(main):001:0> x ||= begin puts('MEH'); false; end
MEH
=> false
irb(main):002:0> x ||= begin puts('MEH'); false; end
MEH
=> false
```